index.bs

```diff
@@ -1483,6 +1561,34 @@ Whenever a [=WebTransport stream=] associated with a {{ReceiveStream}} |stream|
 
 </div>
 
+## `WebTransportReceiveStreamStats` Dictionary ##  {#receive-stream-stats}
+
+The <dfn dictionary>WebTransportReceiveStreamStats</dfn> dictionary includes
+information on stats specific to one {{WebTransportReceiveStream}}.
+
+<pre class="idl">
+dictionary WebTransportReceiveStreamStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long amountReadFrom;
```

Comment:
bytesRead ?

Reply:
Ditto here. E.g.
* With `(await stream.getStats()).bytesReceived` // How many bytes did this _stream_ receive? stream = subject
* With `(await stream.getStats()).bytesRead` // How many bytes were read from this _stream_? stream = object

The hope was that the following would be clearer:
* With `(await stream.getStats()).amountReadFrom` // How many bytes were read from this _stream_?